src/util/GeoSparqlHelpers.h

```diff
@@ -89,6 +92,24 @@ class WktMetricDistGeoPoints {
   }
 };
 
+// A generic operation for all geometric relation functions, like
+// geof:sfIntersects.
+template <SpatialJoinType Relation>
+class WktGeometricRelation {
+ public:
+  ValueId operator()(
+      // TODO<ullingerc> For implementation, use a new appropriate value getter
+      // for geometry literals and points.
+      [[maybe_unused]] const std::optional<GeoPoint>& geoLeft,
+      [[maybe_unused]] const std::optional<GeoPoint>& geoRight) const {
+    AD_THROW(
+        "Geometric relations via the `geof:sf...` functions are not yet "
+        "implemented in QLever. Please refer to the custom `SERVICE qlss:` "
+        "with algorithm `qlss:libspatialjoin` for now. More details can be "
+        "found on the QLever Wiki.");
+  }
```

Comment:
It should be really easy to add a unit test for this throw
in an appropriate Test file (GeoSparqlHelpersTest), 
then you already have the infrastructure set up to test the
`proper` implementations.